Source/WebInspectorUI/UserInterface/Views/ColorPicker.css

```diff
@@ -26,7 +26,8 @@
 .color-picker {
     position: relative;
     width: var(--color-picker-width);
-    height: 236px;
```

Comment:
should we still have this be the `min-height` if the page doesn't have any variables?